pandas/tests/test_frame.py

```diff
@@ -11458,6 +11458,34 @@ def test_var_std(self):
             self.assertFalse((result < 0).any())
             nanops._USE_BOTTLENECK = True
 
+    def test_numeric_only_flag(self):
+        # GH #9201
+        methods = ['sem', 'var', 'std']
+        df = DataFrame(np.random.randn(5, 3), columns=['foo', 'bar', 'baz'])
+        # set one existing entry to a str
+        df.ix[0, 'foo'] = '100'
+
+        df2 = DataFrame(np.random.randn(5, 3), columns=['foo', 'bar', 'baz'])
+        # set one entry in a non-existing column to a str
+        df2.ix[0, 'a'] = '150'
```

Comment:
I mean set the entry to 'a', e.g. make it an object series that has a completely invalid string (and not a string-like number)